pyproject.toml

```diff
@@ -3,22 +3,94 @@ name = "quicknxs"
 description = "Magnetic Reflectivity Reduction"
 dynamic = ["version"]
 requires-python = ">=3.10"
-dependencies = ["mr_reduction==2.5.0"]
 license = { text = "Apache-2.0" }
 keywords = ["neutrons", "quicknxs", "magnetic reflectivity"]
 readme = "README.md"
 
+dependencies = [
+    # Dependencies available from both PyPI and conda
+    "mr_reduction==2.5.0",
```

Comment:
A bunch of these are build dependencies and not runtime dependencies, e.g. `pytest*` and `boa`. Get them out of this section and move them to the appropriate place.